project/src/components/header/header.tsx

```diff
@@ -48,9 +34,9 @@ function Header({authorizationStatus, login,  onLogout}: PropsFromRedux): JSX.El
                       </Link>
                     </li>
                     <li className="header__nav-item" onClick={onLogout}>
-                      <Link className="header__nav-link" to={AppRoute.Main}> {/*TODO возможно потом убрать, так как в тз нет перехода на страницу авторизации*/}
+                      <a className="header__nav-link" >
```

Comment:
`onLogout` должен вызываться на активном элементе. `<li>` считается пассивным и на нем вызывать обработчики некорректно
```suggestion
                      <a className="header__nav-link" onClick={onLogout} >
```